test/controllers/initial.test.js

```diff
@@ -13,6 +13,7 @@ describe("Testing behaviour of verifyToken and generateToken functions", () => {
     const token = generateToken({ username: "this", email: "this@gmail.com" });
     expect(token).to.a("object");
     expect(token).to.have.property("token");
+    console.log(token);
```

Comment:
console logging the token isn't good when it comes to `production`

Reply:
@hervebu Thanks, the issue has been solved.